Makefile

```diff
@@ -44,14 +44,27 @@ integration-test-sumcoins:
 		--db-address "$(STANDARD_REDIS_ADDR)" --db-slot "$(STANDARD_REDIS_DB)" \
 		--encoding "$(STANDARD_ENCODING_TYPE)"
 
-integration-test-sumcoins-python:
+
+generate-python-proto-pkg:
+	test -d tests/integration/sumcoins/build || mkdir tests/integration/sumcoins/build
+	protoc -I=./pkg/types/  --python_out=tests/integration/sumcoins/build ./pkg/types/types.proto
```

Comment:
thx for this though, do I need to install some plugin or does `protoc` by default support python output @xmonader?